### PR TITLE
Clean blazor pages

### PR DESCRIPTION
**Pull request by frederikmnielsen**
Removed some components and pages we don't need

### CODE REVIEW

File: .local/connection_string.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Server=localhost;Database=ProjectBank;User Id=sa;Password=P@55w0rd
```
Comment (frejabv) [inline .local/connection_string.txt:1]: This should probably be removed

File: .local/db_password.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+P@55w0rd
```
Comment (frejabv) [inline .local/db_password.txt:1]: This as well, we might need to look into the gitignore again?

File: ProjectBank/Server/Model/Extensions.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+//         Updated => new NoContentResult(),
+//         Deleted => new NoContentResult(),
+//         NotFound => new NotFoundResult(),
+//         Conflict => new ConflictResult(),
+//         _ => throw new NotSupportedException($"{status} not supported")
+//     };
+
+//     public static ActionResult<T> ToActionResult<T>(this Option<T> option) where T : class
+//         => option.IsSome ? option.Value : new NotFoundResult();
+}
```
Comment (frejabv) [inline ProjectBank/Server/Model/Extensions.cs:16]: Was this not supposed to have been deleted?

**Review by frejabv — CHANGES_REQUESTED** [review]
Looks very nice! But I'm confused about why some of the files are there.

**Review by JacobMoller — APPROVED** [review]

**Review by frejabv — APPROVED** [review]

**Pull request merged**